src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
+
+  if (days >= 7) {
+    return totalCost - 50;
+  }
+
+  if (days >= 3) {
+    return totalCost - 20;
+  }
+
+  return totalCost;
 }
```

Comment:
Lore meant that you should put all numbers into variables. For example: 
`const PRICE = 40;
const DISCOUNT_BIG = 50;`
and so on.

of you have questions, feel free to ask in the chat